tests/nightwatch.conf.js

```diff
@@ -0,0 +1,285 @@
+// Autogenerated by Nightwatch
+// Refer to the online docs for more details: https://nightwatchjs.org/gettingstarted/configuration/
+const Services = {}; loadServices();
```

Comment:
Is this file used at all? It looks like the two local configurations derive from it?

Reply:
This was automatically generated, and I used it as a source for a lot of the things in other two config files, but we ultimately don't need it.

Reply:
OK, then I think we should remove it to reduce the potential for confusion.